### PR TITLE
Don 1079 ii

### PR DESCRIPTION
**Pull request by bdsl**

### CODE REVIEW

File: message-slack.mjs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "Authorization": "Bearer " + env.SLACK_ACCESS_TOKEN
+  },
+  body: JSON.stringify(
+    {
+      "channel": "deployments",
+      "blocks": [{
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": `*${env.CIRCLE_BRANCH}: Donate frontend deployed!* All visitors should see the updated app version within 30 minutes.\n` +
```
Comment (NoelLH) [inline message-slack.mjs:20]: I guess for `develop` we will lose the distinction between Staging & Regression here? Could it use the `env` job parameter that we already pass in for Jira?

**Review by NoelLH — APPROVED** [review]
Looks good! One minor thing inline.

I also wonder if we will later want to think about publishing this in our own orb if we want other apps to do very similar, but that's probably a question for later.

File: message-slack.mjs (hunk 1 above)
Comment (bdsl) [reply message-slack.mjs:20]: Sort of but we haven't been sending these messages for regression in the past, only for staging, and I'm continuing to only show it for staging. I don't think we really need the notifications for regression, it happens at the same time.

**Comment by bdsl** [comment]
> I also wonder if we will later want to think about publishing this in our own orb if we want other apps to do very similar, but that's probably a question for later.

I'm not so keen on orbs, it feels like it ties us in to circleci too much, which isn't a problem in terms of wanting to stop using circle but makes them harder to debug since they can only be run from circle workflows - not as easily afaik from command line even inside a circle executor. If we wanted to publish something we could do via NPM but I don't think the script I've written here is worth publishing separately, may be as good to copy and paste between repos.

**Pull request merged**